VerilogCodingStyle.md

```diff
@@ -2494,6 +2493,68 @@ always_ff @(posedge clk) begin
 end
 ```
 
+### Wildcard import of packages
+
+It is not permitted to use wildcard import syntax, eg. `import other_pkg::*;`
+in general to avoid naming collisions. However, wildcard import is allowed if
+the imported package is placed at the same IP and the import statement is
+placed at the module header or the module body.
+
+&#x1f44d;
+```systemverilog {.good}
+// mod_a_pkg.sv and mod_a.sv are placed at the same directory
```

Comment:
Maybe add a comment here what this syntax is good for and when to prefer it. Effectively: "Packages can be imported in the module declaration if access to unqualified types is needed in the port list."

Reply:
OK added comment and revised the example to show the usage case.